packages/suite/src/components/suite/styled.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const CircleBorder = styled.div`
```

Comment:
Could you rename this file to something less generic?

I would keep here only `CircleBorder` but I suggest to rename it as `IconBorderedWrapper`.

Reply:
https://github.com/everstake/trezor-suite/commit/9d06b0a8da0dc44f40d19f7206507188b57f0e19